docs/vault/guide.md

```diff
@@ -78,6 +215,56 @@ You can decide if your Vault should accept new issue requests by users.
+
+1. Navigate to your Vault dashboard and check the "Locked BTC". This gives you the locked BTC that the Interlay or Kintsugi chain assume the Vault holds. In the example below, the `DOT/IBTC` Vault has `7.85948385` BTC locked.
+![BTC_locked](../_assets/img/vault/btc-locked.png)
+2. Connect to the system running the Bitcoin node for your Vault client. You can list the BTC balance of the `DOT/IBTC` Vault by running the following command on the CLI with the keyname you specified when running the Vault client (from the [installation instructions](vault/installation?id=_5-start-the-vault-client)):
+
+```sh
+bitcoin-cli -rpcwallet=<INSERT_YOUR_KEYNAME, example: 0x0e5aabe5ff862d66bcba0912bf1b3d4364df0eeec0a8137704e2c16259486a71>-DOT-IBTC getbalance
+```
+
+The output will look like
+
+```sh
+$ bitcoin-cli -rpcwallet=0x0e5aabe5ff862d66bcba0912bf1b3d4364df0eeec0a8137704e2c16259486a71-DOT-IBTC getbalance
+> 7.8678
+```
+
+In the example given here, the Vault client is healthy: the Vault client has `7.8678` BTC in its Bitcoin wallet and the Interlay chain assumes that it has `7.85948385` BTC locked. That means, even if the Vault is requested to return all BTC, it will have enough BTC to fulfill those requests.
+
+?> If you are unsure about the Bitcoin wallets or you used an alternative for specifiying the Vault keyfile, you can run `bitcoin-cli listwallets` to see all your Vault wallets. You can then insert each item of the returned rpc wallets to check their balance.
+
+### What if my BTC balance in the Bitcoin wallet is higher than the BTC Locked on the App Dashboard?
+
+In this case, all is well with your Vault. You do not need to do anything.
+
+### What if my BTC balance in the Bitcoin wallet is smaller than the BTC Locked on the App Dashboard?
+
+**What does this mean?**
+
+The BTC balance in your Vault's Bitcoin wallet is too low to fulfill a redeem request that would redeem all of the Vault's BTC locked according to the Interlay or Kintsugi chain.
+
+**What should be done about that?**
+
+You have two options:
+
+1. *Wait and observe*: On redeem requests, users tend to pay more for BTC transaction fees to the Vault than the actual BTC transaction fee that the Vault has to pay. If the difference between the Bitcoin wallet balance and the Interlay/Kintsugi chain balance is quite small, it might be OK to just wait for a couple of redeem request. Ideally, the Vault will have enough BTC in its wallet.
+2. *Insufficient BTC to fulfill a redeem request*: If a user request to redeem more BTC from your Vault than the Vault has in its wallet, the Vault client will show a `BitcoinError ... Insufficient funds...` in the logs. In that case, you need to send BTC from another source to your Vault wallet.
```

Comment:
Should we recommend them to get in touch with us on discord so we can check what happened?

Reply:
yes, very good point.